packages/visx-visx/test/index.test.ts

```diff
@@ -1,120 +1,120 @@
 /* eslint-disable import/namespace */
-import * as visx from '../src';
+import visx from '../src';
```

Comment:
ah I think this is failing the build since there is no default export

```suggestion
import * as visx from '../src';
```